generators/app/templates/README.md

```diff
@@ -10,88 +14,100 @@ This README is intended to help get you started. Definitely update and improve t
+writing it yourself, you can use existing plugins.
 
-[hubot-scripts]: https://github.com/github/hubot-scripts
+Hubot is able to load plugins from third-party `npm` packages. This is the
+recommended way to add functionality to your hubot. You can get a list of
+available hubot plugins on [npmjs.com](npmjs) or by using `npm search`:
+
+    % npm search hubot-scripts panda
+    NAME             DESCRIPTION                        AUTHOR DATE       VERSION KEYWORDS
+    hubot-pandapanda a hubot script for panda responses =missu 2014-11-30 0.9.2   hubot hubot-scripts panda
+    ...
 
-### external-scripts
 
-Hubot is able to load scripts from third-party `npm` package. Check the package's documentation, but in general it is:
+To use a package, check the package's documentation, but in general it is:
 
-1. Add the packages as dependencies into your `package.json`
-2. `npm install` to make sure those packages are installed
-3. Add the package name to `external-scripts.json` as a double quoted string
+1. Use `npm install --save` to add the package to `package.json` and install it
+2. Add the package name to `external-scripts.json` as a double quoted string
 
 You can review `external-scripts.json` to see what is included by default.
 
+[npmjs]: https://www.npmjs.com
+
+### hubot-scripts
+
+Before hubot plugin packages were adopted, most plugins were held in the
+[hubot-scripts][hubot-scripts] package. Some of these plugins have yet to be
+migrated to their own packages. They can still be used but the setup is a bit
+different.
+
+To enable scripts from the hubot-scripts package, add the script name with
+extension as a double quoted string to the `hubot-scripts.json` file in this
+repo.
+
+[hubot-scripts]: https://github.com/github/hubot-scripts
+
 ##  Persistence
 
-If you are going to use the `hubot-redis-brain` package
-(strongly suggested), you will need to add the Redis to Go addon on Heroku which requires a verified
+If you are going to use the `hubot-redis-brain` package (strongly suggested),
+you will need to add the Redis to Go addon on Heroku which requires a verified
 account or you can create an account at [Redis to Go][redistogo] and manually
 set the `REDISTOGO_URL` variable.
 
     % heroku config:add REDISTOGO_URL="..."
 
-If you don't require any persistence feel free to remove the
-`hubot-redis-brain` from `external-scripts.json` and you don't need to worry
-about redis at all.
+If you don't need any persistence feel free to remove the `hubot-redis-brain`
+from `external-scripts.json` and you don't need to worry about redis at all.
 
 [redistogo]: https://redistogo.com/
 
 ## Adapters
 
-Adapters are the interface to the service you want your hubot to run on. This
-can be something like Campfire or IRC. There are a number of third party
-adapters that the community have contributed. Check
-[Hubot Adapters][hubot-adapters] for the available ones.
+Adapters are the interface to the service you want your hubot to run on. For
```

Comment:
"For instance Campfire or IRC" is a sentence fragment. Maybe add to the end of the prior sentence, ie "Adapters are the interface to the service you want your hubot to run on, such as Campfire or IRC".